migration/iD10/oracle/202207141111_IDEMPIERE-1963.sql

```diff
@@ -0,0 +1,620 @@
+;
+
+-- Jul 14, 2022, 11:52:26 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=70,Updated=TO_TIMESTAMP('2022-07-14 11:52:26','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210467
+;
+
+-- Jul 14, 2022, 11:52:30 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=80,Updated=TO_TIMESTAMP('2022-07-14 11:52:30','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210468
+;
+
+-- Jul 14, 2022, 11:53:45 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=90,Updated=TO_TIMESTAMP('2022-07-14 11:53:45','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210469
+;
+
+-- Jul 14, 2022, 11:53:59 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=100,Updated=TO_TIMESTAMP('2022-07-14 11:53:59','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210470
+;
+
+-- Jul 14, 2022, 11:54:27 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=110,Updated=TO_TIMESTAMP('2022-07-14 11:54:27','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210471
+;
+
+-- Jul 14, 2022, 11:55:25 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=120,Updated=TO_TIMESTAMP('2022-07-14 11:55:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210472
+;
+
+-- Jul 14, 2022, 11:55:33 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=130,Updated=TO_TIMESTAMP('2022-07-14 11:55:33','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210473
+;
+
+-- Jul 14, 2022, 11:55:49 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=140,Updated=TO_TIMESTAMP('2022-07-14 11:55:49','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210474
+;
+
+-- Jul 14, 2022, 11:55:55 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=150,Updated=TO_TIMESTAMP('2022-07-14 11:55:55','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=217377
+;
+
+-- Jul 14, 2022, 11:56:10 AM CEST
+UPDATE AD_ViewColumn SET SeqNo=160,Updated=TO_TIMESTAMP('2022-07-14 11:56:10','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_ViewColumn_ID=210475
+;
+
+-- Jul 14, 2022, 12:01:21 PM CEST
+DROP VIEW M_InOut_Candidate_v
+;
+
+-- Jul 14, 2022, 12:01:22 PM CEST
+CREATE OR REPLACE VIEW M_InOut_Candidate_v(AD_Client_ID, AD_Org_ID, Created, Updated, IsActive, C_BPartner_ID, C_Order_ID, DocumentNo, DateOrdered, C_DocType_ID, POReference, Description, SalesRep_ID, M_Warehouse_ID, M_InOut_ID, TotalLines, DocSource, M_InOut_Candidate_v_ID, DeliveryRule) AS SELECT o.ad_client_id AS AD_Client_ID, o.ad_org_id AS AD_Org_ID, o.created AS Created, o.updated AS Updated, o.isactive AS IsActive, o.c_bpartner_id AS C_BPartner_ID, o.c_order_id AS C_Order_ID, o.documentno AS DocumentNo, o.dateordered AS DateOrdered, o.c_doctype_id AS C_DocType_ID, o.poreference AS POReference, o.description AS Description, o.salesrep_id AS SalesRep_ID, l.m_warehouse_id AS M_Warehouse_ID, NULL::numeric AS M_InOut_ID, sum((l.qtyordered - l.qtydelivered) * l.priceactual) AS TotalLines, 'O'::text AS DocSource, o.c_order_id AS M_InOut_Candidate_v_ID, o.deliveryrule FROM c_order o
```

Comment:
This threw error in oracle:
ORA-00923: FROM keyword not found where expected

Reply:
I regenerated this line, there was a missing alias (AS DeliveryRule), I'm not sure if that was the problem. I added the missing alias in the postgresql script too.